include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -197,23 +222,36 @@ namespace NovelRT::Maths
                 *reinterpret_cast<const glm::vec2*>(&point);
         }
 
+        /**
+         * @brief Compare this vector to other vector by using epsilonValue vector as a tolerance for any deviation between the two vectors for comparison
+         * @returns Return true if comparison between this vector and other vector fall within the tolerance set by epsilonValue vector, otherwise false
+         */
```

Comment:
more suggestions! :D
```suggestion
        /**
         * @brief Compares this Geovector2F to another GeoVector2F by using an epsilon valu as a tolerance for any deviation between the two vectors for comparison.
         * @returns Return true if the difference between this vector and other vector falls within the tolerance set by the epsilonValue vector, otherwise false.
         */
```

This also appears to be missing the `@param` descriptors, please add them. Thank you!